backend/cache/cache_upload_test.go

```diff
@@ -160,11 +160,11 @@ func TestInternalUploadQueueMoreFiles(t *testing.T) {
 	minSize := 5242880
 	maxSize := 10485760
 	totalFiles := 10
-	rand.Seed(time.Now().Unix())
+	randInstance := rand.New(rand.NewSource(time.Now().Unix()))
```

Comment:
Why did we need these changed? Did something complain?

Reply:
This came about as part of the linting tests for go1.20. There are a few more similar complaints, which I haven't gotten around to fixing, so I can separate these out and put them all as part of a separate PR if you want.

Reply:
Which lint did you run? The golangci-lint we use as part of the build is passing so it thinks everything is OK now!

Did it fail before those changes?

Reply:
Yeah, govet as part of golangci-lint, but I ran it with go.mod temporarily modified to 1.20 rather than 1.19.

It's only a problem with Go 1.20 where the above function was deprecated: https://pkg.go.dev/math/rand#Seed (https://tip.golang.org/doc/go1.20#math/rand)